core/string/ustring.cpp

```diff
@@ -1951,12 +1892,12 @@ Error String::parse_ascii(const Span<char> &p_range) {
 
 String String::utf8(const char *p_utf8, int p_len) {
 	String ret;
-	ret.parse_utf8(p_utf8, p_len);
+	ret.extend_utf8(p_utf8, p_len);
 
 	return ret;
 }
 
-Error String::parse_utf8(const char *p_utf8, int p_len, bool p_skip_cr) {
+Error String::extend_utf8(const char *p_utf8, int p_len, bool p_skip_cr) {
```

Comment:
Would `append_utf8` be a better name or is **extend** the correct term to use in this domain?

Reply:
Most APIs i interacted with use `append` for single elements and `extend` for multiple elements at once.

Reply:
Note that our own API doesn't use this naming, for example `Array.append_array`, and `Vector.append_array`, so we have an established `append` currently (including bindings for `Array` and `Packed*Array`)

Reply:
Would you suggest changing it? I definitely prefer using two different names but I'm open to it if it means it's more consistent.

Reply:
Alright, that's good enough for me. I'll quickly swap it around to `append_utfx`